From 6dc5a7efb4fa7dc131fe708f9bd12b1dbf30bdd5 Mon Sep 17 00:00:00 2001
From: Ravi Kumar Siddojigari <rsiddoji@codeaurora.org>
Date: Mon, 20 Feb 2017 18:51:27 +0530
Subject: [PATCH] selinux: update the getty.te for addressing the denial

type=1400 audit(17474.349:98): avc:  denied  { read write  pid=1303 comm="agetty"
path="socket:[17094]" dev="sockfs" ino=17094 scontext=system_u:system_r:getty_t:s0-s15:c0.c1023
tcontext=system_u:system_r:init_t:s0-s15:c0.c1023 tclass=unix_stream_socket permissive=1

---
 policy/modules/system/getty.te | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/policy/modules/system/getty.te b/policy/modules/system/getty.te
index c704ddb82..d467e422b 100644
--- a/policy/modules/system/getty.te
+++ b/policy/modules/system/getty.te
@@ -127,3 +127,5 @@ optional_policy(`
 optional_policy(`
 	ppp_domtrans(getty_t)
 ')
+
+init_rw_stream_sockets( getty_t)
